### ISSUE TITLE
Add color control net plz

### ISSUE THREAD

**Issue by xueqing0622** [lllyasviel/ControlNet#146]
Add color control net plz, stable diffusion is very hard to control color,  The Colors easily pollute each other and make mistake.
if can ,Add color control net plz.
Someone can add this to pull requests plz.

**Comment by xiankgx** [comment]
Color control is something img2img in the original SD is about.

**Comment by ninjasaid2k** [comment]
> 

Have you tried combining img2img with Controlnet?

**Comment by xueqing0622** [comment]
No, still in txt2img, just like canny or Scribbles add color theme below, 
Canny or scribbles is very nice to control the shape of image, 
But as I say  stable diffusion is very hard to control color, The Colors easily pollute each other and make mistake.
we still need color control in control net to control the color,

Let me use a picture to explane what I mean:
[https://imgur.com/a/oqxi58M](url)  
just like picture, we have 1, the canny or scribbles mode now. 
And now we need the  new mode like 2 " line and color mode", to control the color for final result' s color, like 3

**Comment by xueqing0622** [comment]
The img2img use control net can do same thing, but 
1 should drag picture twice, 
2 should make line no so strong 
3  must to preprocessor
4  use more gpu

**Comment by jinxixiang** [comment]
I don't know whether the spatial palette mode in T2I adapter can fulfill your requirements. https://github.com/TencentARC/T2I-Adapter

**Comment by jinxixiang** [comment]
> yes , we urgently need a control for color, img2img is not very good for color control because img2img not only influences the color of the output but also influences the geometry and style the lower the strength but we want to leave these last two intact, T2I control doesn't work very well

we make a simple extension for proof of concept. FYI.
https://github.com/jinxixiang/color_controlnet

**Comment by loboere** [comment]
@jinxixiang  interesting, can I use your color model with this Mikubill extension?

**Comment by jinxixiang** [comment]
@loboere Sure, we plan to release a Webui demo later.

**Comment by lsycxyj** [comment]
Any progress?
